examples/gradle-kotlin-dsl/src/main/kotlin/Test.kt

```diff
@@ -1,13 +1,15 @@
 package incorrect
 
-class incorrectname: Exception() {
-    fun INCORRECT_FUNCTION() {
-        throw Exception()
-    }
-
+class IncorrectnameException : Exception() {
     // fun myCommentedFunction() {
     // }
 
     val Incorrect_Val = 5
 
+    /**
+     * @throws Exception
+     */
+    fun incorrectFunction() {
+        throw Exception()
+    }
 }
```

Comment:
Maybe it's worth changing all Test files in examples?

Reply:
did it by mistake, will revert

Reply:
done